dietpi/dietpi-software

```diff
@@ -8966,6 +9010,43 @@ _EOF_
 			pihole -w freegeoip.app --comment 'Used by dietpi-banner and dietpi-vpn to obtain your public IP and location'
 
 		fi
+		
+		software_id=126 # AdGuard Home
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			
+			if [[ ! -f '/opt/AdGuardHome/AdGuardHome.yaml' ]]; then
+				dps_index=$software_id Download_Install 'AdGuardHome.yaml' /opt/AdGuardHome/AdGuardHome.yaml
+				G_CONFIG_INJECT 'password:[[:blank:]]' "  password: $(htpasswd -bnBC 10 "" $GLOBAL_PW | tr -d ':\n' | sed 's/$2y/$2a/')" /opt/AdGuardHome/AdGuardHome.yaml
+			fi
+
+			# Service
+			cat << _EOF_ > /etc/systemd/system/AdGuardHome.service
+[Unit]
+Description=AdGuard Home: Network-level blocker
+ConditionFileIsExecutable=/opt/AdGuardHome/AdGuardHome
+After=syslog.target network-online.target
+StartLimitInterval=5
+StartLimitIntervalSec=10
+
+[Service]
+ExecStart=/opt/AdGuardHome/AdGuardHome "-s" "run"
+
+WorkingDirectory=/opt/AdGuardHome
+
+StandardOutput=file:/var/log/AdGuardHome.out
+StandardError=file:/var/log/AdGuardHome.err
```

Comment:
Let's keep STDOUT (default) as long log target, or do you think plain text log files have much benefit over `journalctl -u adguardhome`? The letter must be used anyway to check for systemd/service side failures and service state. This also avoids issues when using a non-root users (which I would prefer) where we'd need to have a subdirectory for log files pre-created.

Reply:
ok I removed the 2 lines. Output is going to STDOUT now